app/presenters/quota_presenter.rb

```diff
@@ -0,0 +1,92 @@
+# Decorator for {Quota} object making it more presentable for the view
+class QuotaPresenter < SimpleDelegator
+  # Size of block in bytes
```

Comment:
I think that many of these methods belong on Quota instead of the presenter.

For example:

```
block_usage_in_bytes
total_block_usage_in_bytes
block_limit_in_bytes
total_block_usage_percent
total_file_usage_percent
```

Those legitimately could be on the Quota object itself. If those are moved to Quota, including BLOCK_SIZE then I question the need for this class.

For example in erb:

```ruby
quota.human_block_usage
```

is not a huge difference from 

```ruby
number_to_human_size(quota.block_usage_in_bytes)
```

though it does look like some of the methods below (human_resource_limit and total_resource_usage_percent) provide a helpful abstraction to keep the view really simple.